### PR TITLE
Add RFC writeup

### PR DESCRIPTION
**Pull request by amyrlam**
Thank you @MelSumner for the awesome writeup! 💯

### CODE REVIEW

File: source/2020-08-14-the-ember-times-issue-160.md [hunk 1]
```diff
@@ -25,14 +26,29 @@ READMORE
...
+
+Once you are ready to write the RFC for your awesome idea, there are a few important steps: 
+
+1. Read through the entire [README](https://github.com/emberjs/rfcs/) so that you understand what your RFC needs to succeed. Sometimes folks will skip over the ["How we teach this"](https://github.com/emberjs/rfcs/blob/master/0000-template.md#how-we-teach-this) section or the ["Alternatives"](https://github.com/emberjs/rfcs/blob/master/0000-template.md#alternatives) section of the template, and both of those are important to include for the RFC to proceed. 
+1. Find a core team member to champion the RFC! All RFCs need a core team champion, so review the [team list](https://emberjs.com/teams/) to see who you should approach about championing your issue. That core team member makes sure that your RFC is discussed in meetings, so it's essential to have one if you want your awesome idea to succeed! 
+
+If you're interested in eventually writing RFCs but don't know where to start, here are some steps you can take on your own:
+
+* Read through the [RFCs that have been accepted](https://emberjs.github.io/rfcs/introduction.html). Look for the patterns, what similarities do they have?
```
Comment (amyrlam) [inline source/2020-08-14-the-ember-times-issue-160.md:48]: @MelSumner I wanted to check that this is the right link for RFCs that have been accepted?

Comment (MelSumner) [reply source/2020-08-14-the-ember-times-issue-160.md:48]: @amyrlam yup looks good!

**Review by MelSumner — APPROVED** [review]
Thanks Amy! 👍

File: source/2020-08-14-the-ember-times-issue-160.md (hunk 1 above)
Comment (amyrlam) [reply source/2020-08-14-the-ember-times-issue-160.md:48]: thank you!

**Pull request merged**